locale/fr/translations.json

```diff
@@ -79,6 +79,8 @@
   "Add choice": "Ajouter une option",
   "Add code playground": "Ajouter une aire de jeux codée",
   "Add emoji": "Ajouter un emoji",
+  "autocompletes from": "autocomplète à partir de", 
+  "autocompletes": "autocomplète",
```

Comment:
This is OK for testing, but we never commit changes to translations directly like this, since they'll just be removed when we next sync translation data from Transifex. So I'd put the changes to locale files in a separate commit that we won't merge.

See https://zulip.readthedocs.io/en/latest/translating/internationalization.html for details.